Packs/Use_Case_Builder/Scripts/UseCasePlaybookAutoGenerator/UseCasePlaybookAutoGenerator.py

```diff
@@ -0,0 +1,106 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from typing import Dict, Any
+import traceback, json
```

Comment:
The validations will fail on it. Please split the import into two different imports.